types/IntersectionObserver.d.ts

```diff
@@ -33,10 +33,21 @@ export interface IntersectionObserverProps {
    * @default false
    */
   intersecting?: boolean;
+
+  /**
+   * @default null
+   */
+  observer?: null | Observer;
+
+  /**
+   * @default false
+   */
+  once?: null | boolean;
 }
 
 export default class IntersectionObserver extends SvelteComponentTyped<
   IntersectionObserverProps,
+  { intersect: CustomEvent<Entry> },
   { observe: CustomEvent<Entry> },
-  { default: { intersecting: boolean; entry: Entry } }
+  { default: { intersecting: boolean; entry: Entry, observer: Observer, once: boolean; } }
```

Comment:
```suggestion
  { default: { intersecting: boolean; entry: Entry; observer: IntersectionObserver; } }
```

The `once` prop isn't passed to the default slot. Assuming this is a typo

Reply:
Yes, this was also leftover stuff I meant to remove.